src/Render/Program.cs

```diff
@@ -48,9 +48,10 @@ internal static class Program {
 
 			var text = await File.ReadAllTextAsync( filename );
 
+			var context = new DocumentContext( "test" );

```

Comment:
How will we get the name of the repo for this? Should we add an argument to the program for which docroot we are working in?

Reply:
I think that sounds reasonable. Your entrypoint can probably get it from an env var? Check the GitHub Actions docs.... there should be something there.

Reply:
https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables